src/app/demos/date-range-picker/date-range-picker-demo.component.html

```diff
@@ -0,0 +1,117 @@
+<div
+  class="date-range-demo"
+>
+  <form
+    id="screenshot-date-range-picker"
+    novalidate
+    [formGroup]="reactiveForm"
+  >
+    <div class="sky-form-group">
+      <sky-date-range-picker
+        formControlName="lastDonation"
+        label="Last donation"
+        [calculatorIds]="calculatorIds"
+        [dateFormat]="dateFormat"
+      >
+      </sky-date-range-picker>
+
+      <ng-container *ngIf="reactiveRange.errors && reactiveRange.touched">
+        <div *ngIf="reactiveRange.errors.skyDateRange?.errors.endDateBeforeStartDate"
+          class="sky-error-label"
+        >
+          End date must be before start date.
+        </div>
+        <div *ngIf="reactiveRange.errors.skyDate"
+          class="sky-error-label"
+        >
+          Invalid date entered.
+        </div>
+      </ng-container>
+    </div>
+  </form>
+
+  <h3>
+    Form control statuses
+  </h3>
+
+  <table>
```

Comment:
Any reason these values are centered? I'm guessing that's the default `th` style...

Reply:
It is the default style of `th`, yes. I'm okay with it if @blackbaud-johnly is.

Reply:
I notice we're doing this on other PRs. Let's just disregard for now. Its super minor anyways.